spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/client/RestTemplateBuilder.java

```diff
@@ -64,6 +66,10 @@
 
 	private final String rootUri;
 
+	private final Duration readTimeout;
```

Comment:
That's not what I have in mind, please keep your change to a minimum. I expect two new methods passing a `Duration` to the existing customizers with the deprecated methods delegating to them.